sdm/libs/hwc2/hwc_display.cpp

```diff
@@ -1788,11 +1788,15 @@ int HWCDisplay::SetDisplayStatus(DisplayStatus display_status) {
   switch (display_status) {
     case kDisplayStatusResume:
       display_paused_ = false;
+      status = INT32(SetPowerMode(HWC2::PowerMode::On, false /* teardown */));
```

Comment:
please remove false flag. or you can cherry-pick this one:)
https://source.codeaurora.org/quic/la/platform/hardware/qcom/display/commit/sdm/libs/hwc2/hwc_display.cpp?h=LA.UM.7.8.r4-01000-SDM710.0&id=9b6b2f131e03dabb9941fa4776540481bc1c7b3f

Reply:
Done, cherry-picked as https://github.com/sonyxperiadev/hardware-qcom-display/pull/22/commits/7da54855b89a67a2f43514f62bedce49f1a4b3c3